packages/EasySchedule/src/Bridge/Doctrine/CommandExecutedEventListener.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasySchedule\Bridge\Doctrine;
+
+use Doctrine\ORM\EntityManagerInterface;
+use EonX\EasySchedule\Events\CommandExecutedEvent;
+
+final class CommandExecutedEventListener
+{
+    public function __construct(private readonly EntityManagerInterface $entityManager)
```

Comment:
This will work only for applications that have a single DB connection, as soon as you have more than 1 connection then you have more than 1 entity manager and we should clear all of them otherwise we will face issues. We need to account for this.

Best way to do that is to inject the manager registry and loop through the list of managers and if it's an entity manager then call clear on it 😉

Reply:
Fixed